### PR TITLE
feat(icons): add `coffee-filter` icon

### PR DESCRIPTION
**Pull request by adamplesnik**
## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description

Icon for the so-called alternative coffee preparation: in this case the filter, or v60, chemex.

### Icon use case 

It would be convenient to have an icon for the so-called alternative coffee preparation: in this case the filter, or v60, chemex.

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [ ] I've based them on the following Lucide icons: 
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [ ] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by adamplesnik** [comment]
Thanks for a great tip! I updated the icon, feedback is welcome.

**Comment by jguddas** [comment]
icon (58)

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22m14+11+4.17-7.51A1+1+0+0+0+17.3+2H6.7a1+1+0+0+0-.87+1.49L10+11%22+/%3E%3Cpath+d=%22M15.281+11a1+1+0+0+1+.948.681L19.2+20.52a1+1+0+0+1-.87+1.48H5.69a1+1+0+0+1-.89-1.48l2.971-8.839A1+1+0+0+1+8.719+11%22+/%3E%3Cpath+d=%22M5+11h14%22+/%3E%3Cpath+d=%22M5.5+18.379A6.472+6.472+0+0+1+12+17.5a6.47+6.47+0+0+0+6-.532%22+/%3E">icons<br/>Open lucide studio</a>

**Comment by jguddas** [comment]
I don't like the execution of the handle but, maybe it's still worth an idea.

icon (60)


<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22m14+11+4.17-7.51A1+1+0+0+0+17.3+2H6.7a1+1+0+0+0-.87+1.49L10+11%22+/%3E%3Cpath+d=%22M15.281+11a1+1+0+0+1+.948.681L19.2+20.52a1+1+0+0+1-.87+1.48H5.69a1+1+0+0+1-.89-1.48l2.971-8.839A1+1+0+0+1+8.719+11%22+/%3E%3Cpath+d=%22M19.2+20.52A1+1+0+0+0+18+14h-1%22+/%3E%3Cpath+d=%22M5+11h14%22+/%3E%3Cpath+d=%22M5.5+18.379A6.472+6.472+0+0+1+12+17.5a6.47+6.47+0+0+0+6-.532%22+/%3E">icons<br/>Open lucide studio</a>

File: icons/coffee-filter.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "adamplesnik"
```
Comment (jguddas) [inline icons/coffee-filter.json:4]:
```suggestion
    "adamplesnik",
    "karsa-mistmere",
    "jguddas"
```

**Comment by ericfennis** [comment]
@adamplesnik I doubt the usecase a bit. We need a real-life usecase. Instead of the usecase what it represents. 
Otherwise I recommend to add the icon to [lucide lab](https://github.com/lucide-icons/lucide-lab)

**Comment by adamplesnik** [comment]
> @adamplesnik I doubt the usecase a bit. We need a real-life usecase. Instead of the usecase what it represents. Otherwise I recommend to add the icon to [lucide lab](https://github.com/lucide-icons/lucide-lab)

I don't mind moving it there, thank you for your feedback.

**Comment by ericfennis** [comment]
Should be moved to [Lucide lab](https://github.com/lucide-icons/lucide-lab) due to a lack of real-life use cases.

**Pull request closed without merging**